.github/workflows/ci.yml

```diff
@@ -6,7 +6,7 @@ on:
     branches:
       - '**'
 env:
-  NODE_VERSION: 16.10.0
+  NODE_VERSION: 18.0.0
```

Comment:
Use latest node version, https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V18.md

Reply:
wth.. I was looking for it but I couldn't find it...!